charts/prometheus-blackbox-exporter/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v2
 description: Prometheus Blackbox Exporter
 name: prometheus-blackbox-exporter
-version: 8.6.1
+version: 8.6.2
```

this should be at least be a minor version change as a new feature is introduced.
```suggestion
version: 8.7.0
```

as it seems the old way to configure it is still supported?
otherwise major version upgrade would be needed.